ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/activity-picklist-remote-auto-complete-options.component.ts

```diff
@@ -1,20 +1,22 @@
-import { Component, Inject, Input, OnInit, SimpleChanges } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 import {
     BehaviorSubject,
     debounceTime,
     distinctUntilChanged,
     map,
     Observable,
-    of,
     switchMap,
-    tap,
 } from 'rxjs';
-import { ActivityPicklistNormalizedGroup } from '../../../models/activity/activityPicklistNormalizedGroup';
 import { ActivityPicklistOption } from '../../../models/activity/activityPicklistOption';
 import { NGXTranslateService } from '../../../services/internationalization/ngxTranslate.service';
 import { ActivityServiceAgent } from '../../../services/serviceAgents/activityServiceAgent.service';
 import { BaseActivityPicklistQuestion } from './baseActivityPicklistQuestion.component';
 
+interface SplittedData {
+    plOptions: ActivityPicklistOption[];
+    otherOption: ActivityPicklistOption;
+}
+
```

Comment:
Le't rename this something like CategorizedPicklistOptions
instead of plOptions, let's call it currentAutoCompleteOptions .
otherOption, let's call it userEnteredStringOption

Reply:
Sounds good, renamed